MidasCivil_Adapter/Convert/ToBHoM/Properties/ToProfile.cs

```diff
@@ -45,19 +45,27 @@ public static IProfile ToProfile(List<string> sectionProfile, string shape, stri
                     double webSpacing = System.Convert.ToDouble(sectionProfile[4]).LengthToSI(lengthUnit);
                     double webThickness = System.Convert.ToDouble(sectionProfile[2]).LengthToSI(lengthUnit);
                     double corbel;
-                    if (System.Math.Abs(width / 2 - webSpacing / 2 - webThickness / 2) < oM.Geometry.Tolerance.Distance)
+                    if (webSpacing < oM.Geometry.Tolerance.Distance)
+                    {
+                        corbel = (width / 2 - webThickness / 2).LengthToSI(lengthUnit);
+                    }
+                    else if (System.Math.Abs(width / 2 - webSpacing / 2 - webThickness / 2) < oM.Geometry.Tolerance.Distance)
```

Comment:
```suggestion
                    if (webSpacing < oM.Geometry.Tolerance.Distance || System.Math.Abs(width / 2 - webSpacing / 2 - webThickness / 2) < oM.Geometry.Tolerance.Distance)
                        corbel = 0;
```
- If the web spacing is 0 in MidasCivil - then there are no corbels. See the comparison between the MidasCivil section and the BHoM section below. As it stands it is still assigning corbels where they do not belong.
- Lack of brackets is style choice, but if you remove them, do it for the `else if` statement below too please.


![image](https://user-images.githubusercontent.com/30078587/122640672-bb6ece00-d0f8-11eb-9c09-e2d8bfbe1a1b.png)
![image](https://user-images.githubusercontent.com/30078587/122640677-c0338200-d0f8-11eb-859f-63b363e432d8.png)